test/wdio/listen-jsx/cmp-root.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { h, Fragment } from '@stencil/core';
```

Comment:
Non-blocking - Fragment isn't used and can be deleted

```suggestion
import { h } from '@stencil/core';
```